mito-ai/mito_ai/handlers.py

```diff
@@ -60,7 +54,7 @@ def initialize(self, llm: OpenAIProvider) -> None:
         super().initialize()
         self.log.debug("Initializing websocket connection %s", self.request.path)
         self._llm = llm
-        self.full_message_history: list[ChatCompletionMessageParam] = []
+        self.message_history = GlobalMessageHistory(llm)
```

Comment:
Move this back to not being a class variable so there is one lock manager per backend, not per websocket